quarkus/tests/integration/src/test/java/org/keycloak/it/cli/dist/LoggingDistTest.java

```diff
@@ -168,10 +169,18 @@ void syslogHandler(CLIResult cliResult) {
 
     @Test
     @Launch({"start-dev", "--log-console-level=wrong"})
+    @DryRun
     void wrongLevelForHandlers(CLIResult cliResult) {
         cliResult.assertError("Invalid value for option '--log-console-level': wrong. Expected values are (case insensitive): off, fatal, error, warn, info, debug, trace, all");
     }
 
+    @Test
+    @Launch({"start-dev", "--log-level-org.keycloak=wrong"})
```

Comment:
AFAIK, the `--log-level-"org.keycloak"=wrong` should work as well. Would it be possible to add some test case for it and also mention it in the docs? In order to mimic Quarkus behavior with named props. @vmuzikar WDYT?

Reply:
@mabartos Thanks for the review!

Even though the format with quotes work it's a "byproduct" of SmallRye, so probably no need to test it. The quotes don't bring any benefit and are not really required for our use case. Quarkus need the quotes as they are required for their [mapping](https://github.com/quarkusio/quarkus/blob/0f882426c5a4bbe57eb37ede66c241771b07a135/core/runtime/src/main/java/io/quarkus/runtime/logging/LogRuntimeConfig.java#L96-L98). Also we're not really aligned with Quarkus anyway e.g. in the env var format where they need double underscore (`__`) to mimic the quote.

Reply:
Thanks for the additional info. Works for me.